### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by nseibert**
Add functionality for a better experience with Roon over MQTT

### CODE REVIEW

File: custom_components/multiroom-mqtt-mediaplayer/media_player.py [hunk 1]
```diff
@@ -271,6 +283,8 @@ def __init__(self, hass, config):
         self._supported_features = 0
 
         self._volume = 0.0
+        self._minvolume = -80.0
```
Comment (jonaseickhoff) [inline custom_components/multiroom-mqtt-mediaplayer/media_player.py]: Would be better not to change default behavior. Default would be from minvolume = 0 to maxvolume = 100

**Review by jonaseickhoff — CHANGES_REQUESTED** [review]
Thanks for your PR. Can you change default values for min max to 0 - 100 so that default behavior is same?

File: custom_components/multiroom-mqtt-mediaplayer/media_player.py (hunk 1 above)
Comment (nseibert) [reply custom_components/multiroom-mqtt-mediaplayer/media_player.py]: Ah yes, thank you for checking! It’s a left over from my first version with static values from my setup

**Review by jonaseickhoff — APPROVED** [review]
Thank you!

**Pull request merged**